omen2/omen.py

```diff
@@ -134,6 +137,14 @@ def validate_table(self, name, tab):
                 if fd.name == pk[0]:
                     tab.allow_auto = fd.autoinc
 
+    def bootstrap_row_type(self, tab):
+        bases = getattr(tab, "__orig_bases__", None)
```

Comment:
docstring here:  this infers the row type if you use the generic `Table[RowType]` type decorator.   which is cool, but unclear here.